coconut-rs/src/lib.rs

```diff
@@ -41,9 +45,22 @@ pub use scheme::verification::verify_credential;
 pub use scheme::BlindedSignature;
 pub use scheme::Signature;
 pub use scheme::SignatureShare;
+pub use traits::Base58;
 
 pub type Attribute = Scalar;
 
+impl Bytable for Attribute {
+    fn to_byte_vec(&self) -> Vec<u8> {
+        self.to_bytes().to_vec()
+    }
+
+    fn from_byte_slice(slice: &[u8]) -> Self {
```

Comment:
Given this, I think the trait should rather include a `try_from_byte_slice` (and an associated Error type) because this operation can clearly fail